primitives/contract/src/mapping_token_factory/s2s.rs

```diff
@@ -34,13 +34,13 @@ pub struct Sub2SubMappingTokenFactory;
 impl Sub2SubMappingTokenFactory {
 	/// encode confirm burn and remote unlock deliver message function
 	pub fn encode_confirm_burn_and_remote_unlock(
-		message_id: Vec<u8>,
+		message_nonce: u64,
```

Comment:
Request to revert and keep using message_id instead, for security concerns, and the possibility of chaing land id or supporting multiple lane in app contract side. See `confirmBurnAndRemoteUnlock`